cdap-integration-test/src/test/java/co/cask/cdap/test/IntegrationTestBaseTest.java

```diff
@@ -40,4 +49,26 @@ public void testFlowManager() throws Exception {
     flowManager.stop();
   }
 
+  @Test
+  public void testDeployApplicationInNamespace() throws Exception {
+    Id.Namespace namespace = new Id.Namespace ("Test1");
+    createNamespace(new NamespaceMeta.Builder().setName(namespace).build());
```

Comment:
Probably unrelated to this PR. But why `createNamespace` cannot take `Id.Namespace` but requires a `NamespaceMeta` instead?